packages/http-client-java/generator/http-client-generator-core/src/main/java/com/microsoft/typespec/http/client/generator/core/Javagen.java

```diff
@@ -254,7 +254,7 @@ protected JavaPackage writeToTemplates(CodeModel codeModel, Client client, JavaS
             javaPackage.addPackageInfo(packageInfo.getPackage(), "package-info", packageInfo);
         }
 
-        if (settings.isDataPlaneClient()) {
+        if (settings.isDataPlaneClient() || settings.isUnbranded() || settings.isAzureV2()) {
```

Comment:
Will this affect mgmt(though no mgmt v2 at present)?

Reply:
mgmt should be via `FluentGen.handleTemplate`, so this code is not shared.

Reply:
I see.